synapse/replication/slave/storage/devices.py

```diff
@@ -42,14 +42,19 @@ def __init__(self, db_conn, hs):
 
     def stream_positions(self):
         result = super(SlavedDeviceStore, self).stream_positions()
-        result["device_lists"] = self._device_list_id_gen.get_current_token()
+        result["user_signature"] = result[
+            "device_lists"
+        ] = self._device_list_id_gen.get_current_token()
```

Comment:
This is kind of ugly, but it's what black came up with. :man_shrugging: 

Alternatively, I could set result["user_signature"] and result["device_lists"] as separate statements.